modules/oceanbase/src/main/java/org/testcontainers/oceanbase/OceanBaseCEContainer.java

```diff
@@ -0,0 +1,131 @@
+package org.testcontainers.oceanbase;
+
+import org.apache.commons.lang3.StringUtils;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * Testcontainers implementation for OceanBase Community Edition.
+ * <p>
+ * Supported image: {@code oceanbase/oceanbase-ce}
+ * <p>
+ * Exposed ports:
+ * <ul>
+ *     <li>SQL: 2881</li>
+ *     <li>RPC: 2882</li>
+ * </ul>
+ */
+public class OceanBaseCEContainer extends JdbcDatabaseContainer<OceanBaseCEContainer> {
+
+    static final String NAME = "oceanbasece";
+
+    static final String DOCKER_IMAGE_NAME = "oceanbase/oceanbase-ce";
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse(DOCKER_IMAGE_NAME);
+
+    private static final Integer SQL_PORT = 2881;
+
+    private static final Integer RPC_PORT = 2882;
+
+    private static final String SYSTEM_TENANT_NAME = "sys";
+
+    private static final String DEFAULT_TEST_TENANT_NAME = "test";
+
+    private static final String DEFAULT_USERNAME = "root";
+
+    private static final String DEFAULT_PASSWORD = "";
+
+    private static final String DEFAULT_DATABASE_NAME = "test";
+
+    private String tenantName = DEFAULT_TEST_TENANT_NAME;
+
+    private String driverClassName = "com.mysql.cj.jdbc.Driver";
+
+    public OceanBaseCEContainer(String dockerImageName) {
+        this(DockerImageName.parse(dockerImageName));
+    }
+
+    public OceanBaseCEContainer(DockerImageName dockerImageName) {
+        super(dockerImageName);
+        dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
+
+        addExposedPorts(SQL_PORT, RPC_PORT);
```

Comment:
Should we rely on a `LogWaitStrategy` for `boot success!`?

Reply:
The LogWaitStrategy for boot success will wait not only the database but also  other tools like obagent and  obconfigservier, and it's better to use it for non-jdbc test cases.